src/enclave/ServiceProvider/CMakeLists.txt

```diff
@@ -13,16 +12,21 @@ set(SOURCES
   iasrequest.cpp
   sp_crypto.cpp)
 
+link_directories(/opt/openenclave/lib/openenclave/enclave)
```

Comment:
Should we use `$ENV{OE_SDK_PATH}` rather than hardcoding this path?

Reply:
Addressed.